crates/bevy_ecs/src/schedule/executor/multi_threaded.rs

```diff
@@ -506,9 +511,11 @@ impl MultiThreadedExecutor {
                 #[cfg(feature = "trace")]
                 drop(system_guard);
                 if res.is_err() {
+                    println!("Exclusive system `{}` panicked!", &*system.name());
```

Comment:
Doesn't this branch only apply to `apply_system_buffers`?

Reply:
Especially here, I'd like to report the system sets, and ideally the schedule name too.

Reply:
I'm not entirely sure how to get the schedule label/name since it isn't stored in the schedule itself, but in the container of schedules. Maybe `Schedule::new` could be made generic? That way the type name could be stored in the `SystemSchedule`.

Reply:
Oh yeah, I forgot to mention, I fixed that message. It now tells you the system it was applying buffers for when panicking.